<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="combo_mode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Mode 0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mode 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mode 2</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider_speedR">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_speedL">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_reg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Regulator</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Timeout</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_accel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="display_version">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="button_stop">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>97</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="display_error">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_volt">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Error</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceleration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setspeedL">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setspeedR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentL">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current R</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_encoderL">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>245</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoder L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_encoderR">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>275</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoder R</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_voltage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_currentL">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_currentR">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_encoderL">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_encoderR">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_diffL">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_diffR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_speedL">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_speedR">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_accel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSlider" name="slider_both">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>128</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="display_text">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_refresh">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="display_mode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>48</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="display_textEncL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="display_textEncR">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
